cantabular/cantabular_adapter.go

```diff
@@ -0,0 +1,74 @@
+package cantabular
+
```

Comment:
Hmm, I'm not sure if this separate adapter package is necessary or in line with how things are gnerally done elsewhere, especially with what you've done in package models it seems there's now 3 interface layers of abstraction just to get to the main GetPopulationTypes client function.. Maybe we should go through this tomorrow